cmd/deploy/start_test.go

```diff
@@ -98,6 +98,72 @@ func (suite *DeployStartTestSuite) TestDeployStartYAMLSuccess() {
 	suite.Equal(expected.PipelineID, received.DeploymentId, "they should be equal")
 }
 
+func (suite *DeployStartTestSuite) TestDeployStartWithURLSuccess() {
+	expected := &de.StartPipelineResponse{
+		PipelineID: "12345",
+	}
+	err := registerResponder(expected, http.StatusAccepted)
```

Comment:
```suggestion
        suite.NoError(registerResponder(expected, http.StatusAccepted))
```

Reply:
I know the way that you've written the tests matches the rest of the test in the file 🤷

Reply:
I fixed this for the new tests. I will update the whole file in a separate PR.